src/pages/Login.tsx

```diff
@@ -20,6 +21,8 @@ const urls = getUrls();
 const Login = () => {
     useBrowserTitle('browserTitle.login');
 
+    removeGatewayAuthConfig();
```

Comment:
This should also use the useLocalStorage here.

Reply:
Updated.